requirements.txt

```diff
@@ -0,0 +1,4 @@
+# List your controller-side Python package requirements here if exist
```

Comment:
For the reasons explained in https://forum.ansible.com/t/collection-inclusion-requirements-add-requirements-txt-and-bindep-txt/2611/8, this should really **not** be in the collection's root, and there should be an EE config file explicitly pointing to this file *outside* of the root.